core/point/point.go

```diff
@@ -21,3 +23,21 @@ func (p *Point) X() int64 { return p.x }
 
 // Y returns the y-value.
 func (p *Point) Y() int64 { return p.y }
+
+// Two Go-game board position/point translation methods
+// 1. '*Point' immutable int point, to SGF(String of two letters) point
+func (pt *Point) ToSGF() string {
+	sgfX := string(rune((pt.X()) + aValue))
+	sgfY := string(rune((pt.Y()) + aValue))
+	// two lower case letters
+	return sgfX + sgfY
+}
+
+// 3. 'p' SGF(two letter string) Point, to '*Point' immutable Int Point
+func NewFromSGF(sgfPt string) *Point {
```

Comment:
Try running golint: https://github.com/golang/lint

Typically methods have the following pattern for comments:

// ToSGF converts a SGF point to a Point. Example: ab => {0, 1}

Also, you should be clear about whether the returned point is 0-indexed or 1-indexed. I think it probably makes sense to make it 0-indexed.